samples/embedded-auth-with-sdk/pom.xml

```diff
@@ -33,17 +33,10 @@
         <checkstyle-maven-plugin.version>3.1.2</checkstyle-maven-plugin.version>
         <spring.boot.version>2.4.0</spring.boot.version>
         <java.version>1.8</java.version>
-        <okta.sdk.version>0.1.1-beta.6-SNAPSHOT</okta.sdk.version>
+        <com.okta.idx.sdk.version>0.1.0-beta.6</com.okta.idx.sdk.version>
```

Comment:
Changed this to `com.okta..`, but left next `okta.sdk.api.version` as is. Is it necessary?

Reply:
To be consistent with `com.okta.idx.sdk.version` declared in `samples/embedded-sign-in-widget/pom.xml`